.travis.yml

```diff
@@ -0,0 +1,45 @@
+dist: bionic
+jdk: openjdk11
```

Comment:
do we want to support multiple Java versions?

Reply:
JDK8 is ancient, and is unmaintained by Oracle. OpenJDK8 is still maintained. but we can set a compatible build even in newer JDKs:

https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/61f736d2650dd03a954cdd72ac7b2028b93a7ce9/pom.xml#L36-L37

so i think it's fine. i'd rather stick with the Bionic Ubuntu LTS (18.04) and OpenJDK 11 than some older build env.